stackrox/acs-fleet-manager
# ROX-19013 Apply gitops config to centrals

## Description

Reads the gitops configuration, then applies it to the centrals. 
Adds a `centralYAML` field to the `ManagedCentral` struct
Have `fleetshard` use the `centralYAML` (behind feature flag)

#### Accessory
Added a `docker` env because colima doesn't work on my end

## Checklist (Definition of Done)


## Test manual

- Use a hardcoded gitops config
- Add an override patch to modify the central instance
- Verify that the patch is applied to Central
